Compiled from "ValidatorException.java"
public class sun.security.validator.ValidatorException extends java.security.cert.CertificateException {
  public static final java.lang.Object T_NO_TRUST_ANCHOR;
  public static final java.lang.Object T_EE_EXTENSIONS;
  public static final java.lang.Object T_CA_EXTENSIONS;
  public static final java.lang.Object T_CERT_EXPIRED;
  public static final java.lang.Object T_SIGNATURE_ERROR;
  public static final java.lang.Object T_NAME_CHAINING;
  public static final java.lang.Object T_ALGORITHM_DISABLED;
  public static final java.lang.Object T_UNTRUSTED_CERT;
  public sun.security.validator.ValidatorException(java.lang.String);
  public sun.security.validator.ValidatorException(java.lang.String, java.lang.Throwable);
  public sun.security.validator.ValidatorException(java.lang.Object);
  public sun.security.validator.ValidatorException(java.lang.Object, java.security.cert.X509Certificate);
  public sun.security.validator.ValidatorException(java.lang.Object, java.security.cert.X509Certificate, java.lang.Throwable);
  public sun.security.validator.ValidatorException(java.lang.String, java.lang.Object, java.security.cert.X509Certificate);
  public sun.security.validator.ValidatorException(java.lang.String, java.lang.Object, java.security.cert.X509Certificate, java.lang.Throwable);
  public java.lang.Object getErrorType();
  public java.security.cert.X509Certificate getErrorCertificate();
  static {};
}
